.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+on: 
+  push:
+    branches:
+      - '*'
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - name: Yarn install
+        uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: install
+      - name: Yarn Test
+        uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: test
+
+  publish-npm:
+    needs: test
+    runs-on: ubuntu-18.04
+    if: startsWith(github.ref, 'refs/tags/') 
+    steps:
+      - uses: actions/checkout@v1
+      - name: Yarn build
+        uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: build
```

Comment:
locally `yarn build` fails for me with:
```
$ yarn build
yarn run v1.12.3
$ tsc
/bin/sh: 1: tsc: not found
```
but passes after `yarn install`. AFAIK different jobs are different VMs though, so won't have `node_modules`?